### DISCUSSION TITLE
Supporting for custom raw.githubusercontent.com url

### DISCUSSION THREAD

**Discussion by TomsProject** [upptime/upptime#373] [Ideas] [answered]
You know, there is a little bit difficult accessing GitHub in mainland China, so I'd like to change the URL raw.githubusercontent.com into cdn.jsdelivr.net/gh, but I found there are no such settings in the file: .upptimerc.yml. So I hope you can support this as well.

**Comment by dragon-fish** [comment]
Encountered similar issue. I'd wish to set custom endpoint for `raw.githubusercontent.com`.

**Comment by GammaPi** [comment, marked as answer]
Fixed in https://github.com/upptime/status-page/pull/444
